doc/.readthedocs.yaml

```diff
@@ -14,6 +14,7 @@ build:
   jobs:
     pre_install:
       - pip install gitpython pyyaml
+      - git fetch --unshallow || true
```

Comment:
what is the `|| true` bit for by the way? Seems like we would want to fail if that git command fails?